src/util/decorators/Hidden.tsx

```diff
@@ -1,9 +1,21 @@
-export const HIDDEN = 'hidden';
+export const HIDDEN = {
+  DEFAULT: 'hidden',
+  EditableInTable: 'EditableInTable',
+};
 
 export const Hidden = ({
```

Comment:
@deye9 can we please separate this out and have 2 annotations. Lets leave @Hidden() alone as it was before, so that anywhere that has @Hidden(), that field will just be hidden everywhere (in form and in table) as we currently have with the `updatedAt` and `createdAt`. Right now, I think in the form, we still have this:
```
const hideInTable = Reflect.getMetadata(HIDDEN, instance as any, key);

      if (!hideInTable) {
        schema.push(getSchemaForInstanceAndKey(instance, key, requiredFields));
      }
```
so I think that this may have an issue since HIDDEN is now an object and now a string. 

Lets leave @Hidden as it was, and simply create a new annotation for `@HiddenInTable(hiddenOnlyWhenEditing = true)` annotation. This way, if @HiddenInTable() is applied to a field, then it will be hidden in the table in both editing and showing states. If the field is annotated with `@HiddenInTable(true)` then the field is only hidden when the record is being edited. 

I think splitting these out may be helpful, because most likely we will want to have a way to hide things specifically in table only, or in form only, or in both... and it will probably be easier to have multiple annotations to support that.

Reply:
Implemented